test/e2e/set/scyllacluster/scyllacluster_auth.go

```diff
@@ -117,7 +119,18 @@ var _ = g.Describe("ScyllaCluster authentication", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
-		o.Expect(hosts).To(o.ConsistOf(getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)))
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+
+		oldHosts := hosts
+		oldHostIDs := hostIDs
+		hosts, hostIDs, err = utils.GetBroadcastRPCAddressesAndUUIDs(ctx, f.KubeClient().CoreV1(), sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(hosts).To(o.HaveLen(len(oldHosts)))
+		o.Expect(hostIDs).To(o.ConsistOf(oldHostIDs))
```

Comment:
won't this be different for PodIPs and generally depending on the exposure type?
I think we don't need the check here, just the test that explicitly replace nodes

Reply:
> won't this be different for PodIPs and generally depending on the exposure type?

It shouldn't influence the UUIDs of the nodes. This was done in an effort to avoid depending on the exposure type.

> I think we don't need the check here, just the test that explicitly replace nodes

I tried not to reduce coverage and this (and almost every other) test checks that the hosts didn't change after a rollout.